app/handlers/base.py

```diff
@@ -152,6 +153,33 @@ def bulk_verify(mode, collection, accessor):
 
 
 class BaseHandler(PSABaseHandler):
+    @contextmanager
+    def Session(self, use_auto_verify=True):
```

Comment:
Can we change this to just `verify`?

Reply:
I called the function `verify`. I can do `auto_verify` if that's better?

Reply:
I don't know the advantage of making the keyword longer. `verify` as a keyword seems to indicate perfectly well what is going to happen.

I don't think we should "auto" verify---just verify on request.

Reply:
So I can make the input name `verify` and also the function on the session called `verify`. I will just rename the function definition to something like `run_verification` which is never exposed to the user, so it should be ok.

Reply:
But why not also expose `verify` to the user?

Reply:
Yes, I will still name it `session.verify` and the control boolean will be named `session.use_auto_verify` but when inputting the function the parameter will be called just `verify`.

Reply:
Ah! For storing internal state, it is customary to prefix with an underscore: `session._verify`.

Reply:
That's fine in general, but I feel that would be confusing as there will be a method called `verify`. Let me push another commit and we'll see if it looks ok.